src/components/Header.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+const Header = () => (
+  <div className="container">
+    <h1 className="title">Math Magician</h1>
+    <nav className="Nav">
+      <ul className="flex">
+        <li className="link">
+          <Link className="link" to="/Home">Home</Link>
+        </li>
```

Comment:
- [ ] Don't forget to change this route to make it work properly.

```suggestion
        <li className="link">
          <Link className="link" to="/">Home</Link>
        </li>
```

💪 🥇

Reply:
Noted @BrianSammit I will not forget thanks!